fables/constants.py

```diff
@@ -15,3 +15,8 @@
 >>> fables.detect('a_really_big_csv.csv')
 """
 MAX_FILE_SIZE = 1024 ** 3  # bytes -> 1 GB
+
+# Number of bytes used to determine a mimetype from an io-stream. 30k was
+# found to accurately set mimetypes as a result of fables tests. User can
+# reset this var the same as MAX_FILE_SIZE.
+NUM_OF_BYTES_FOR_MIMETYPE = 30000
```

Comment:
Could we add a newline here (see the github warning)? Fables has used black too which should automatically add this.

A non-blocking nit to change or not change at your discretion. Do you feel like `NUM_BYTES*` or `NUMBER_OF_BYTES*` read cleaner than `NUM_OF_BYTES` and a that postfix of `_DETECTION` makes the intent more precise? I.e. `NUM_BYTES_FOR_MIMETYPE_DETECTION`?

Reply:
Aw I missed @jbcooley7 already commenting on the newline.

Reply:
On the naming convention, I wanted to make it clear since it was defined without context in a constants file. I like NUM_BYTES_FOR_MIMETYPE_DETECTION since that makes it clear what is being defined, and for what purpose and in what context. Will update along with trailing space.